src/navbar/navbar.scss

```diff
@@ -1,9 +1,6 @@
 /* Stuff to get the nav to collapse at 991px (when the menu collapses)
  * Taken from http://stackoverflow.com/a/36289507/995325 */
 
-$dot-color: #fff;
-@import "~three-dots/sass/three-dots";
```

Comment:
We can probably remove the `three-dots` package now (if you haven't already).

Reply:
Done

Reply:
Done